### PR TITLE
Present first view according to user setting

### PR DESCRIPTION
**Pull request by akaroml**
Signed-off-by: Rome Li <rome.li@microsoft.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,7 +61,25 @@
...
+            "string"
+          ],
+          "enum": [
+            "auto",
+            "overview",
+            "gettingStarted"
+          ],
+          "default": "auto",
+          "description": "Sets the default view which is presented during the first use.",
+          "scope": "window"
```
Comment (jdneo) [inline package.json]: Any thinking about setting the scope to `window` here?

Comment (akaroml) [reply package.json]: Windows (instance) specific settings which can be configured in user, workspace, or remote settings.

By doing so, users are able to choose the level of impact of this setting, and it sounds appropriate to me. Other thoughts?

Comment (jdneo) [reply package.json]: To me, it seems more like a `application` scope setting. When will the user needs to set different first view for different workspaces?

Comment (akaroml) [reply package.json]: There could be more first UX views targeting a different audience. E.g. we want to present a view for Spring Boot developers. We provide a sample workspace and set the default view to Spring Getting Started, and this option allows us to do that at the workspace level.

Comment (jdneo) [reply package.json]: I see

**Pull request merged**